packages/terra-site/src/examples/theme-provider/DefaultThemeProvider.jsx

```diff
@@ -30,6 +30,13 @@ class DefaultThemeProvider extends React.Component {
           <select value={this.state.theme} onChange={this.handleThemeChange}>
             <option value="">Default</option>
             <option value="cerner-mock-theme">Mock Theme</option>
+            <option value="terra-consumer-theme-concept1">terra-consumer-theme-concept1</option>
```

Comment:
good catch, originally the button theme example was added here and then moved to button page. Will clean up